### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by kumarabd**
Init Meshsync

### CODE REVIEW

**Comment by leecalcote** [comment]
Operator SDK not Kubebuilder, right?

File: controllers/deployment.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package controllers
```
Comment (nitishm) [inline controllers/deployment.go:1]: This file can be removed I suppose

**Review by nitishm — APPROVED** [review]
LGTM

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Ready to merge?

**Comment by leecalcote** [comment]
Gents, let’s avoid these massive PRs. They are too difficult to review. Ready to merge?

**Comment by kumarabd** [comment]
> Gents, let’s avoid these massive PRs. They are too difficult to review. Ready to merge?

@leecalcote  This one is not stable yet to work, and I saw meshery using the existing operator. That's the reason we are operating on a single PR.

**Comment by kumarabd** [comment]
We can merge too, no issue there @leecalcote

**Pull request merged**